### ISSUE TITLE
EDA (Exploratory Data Analysis) ready for beta testing

### ISSUE THREAD

**Issue by reginaastri** [swirldev/swirl_courses#125]
The course corresponding to Roger Peng's DSS course on exploratory data analysis is ready for beta testing.

**Comment by DeepNought** [comment]
Using Windows 7, RStudio ver. 0.98.1091

Lesson 3 - Graphics Devices in R

When I get to this part:

| The second way to create a plot is to send it to a file device. Depending on the type of plot
| you're making, you explicitly launch a graphics device, e.g., a pdf file. Type the command
| pdf(file="myplot.pdf") to launch the file device. This will create the pdf file myplot.pdf in your
| working directory.

and enter the command as instructed, I get this:

> pdf(file="myplot.pdf")
Error in parse(text = correct_expression) : 
  <text>:1:10: unexpected INCOMPLETE_STRING
1: pdf(file="myplot.pdf)
             ^

| Leaving swirl now. Type swirl() to resume.

Even when swirl() itself enters the command after a skip(), same result.  The pdf file is created in the working directory, but my pdf reader doesn't open it, complaining that it is corrupted. Unable to proceed with the lesson beyond this point.

I was able to complete lessons 1 and 2.

**Comment by reginaastri** [comment]
Hi DeepNought.

Thanks for pointing this out. There was a missing quotation mark which I just fixed. Please re-install the course and try it again. Also, you probably won't be able to look at the file until you close the device which, I believe, happens near the end of  the lesson.

Sorry for the inconvenience. I don't know how this slipped past me.

Gina

**Comment by ncarchedi** [comment]
@reginaastri I'm sorry I haven't had a chance to work through this new course yet. I'm very excited about it, though, since I think it's addressing the most obvious gap in our current swirl content -- data visualization. Will pull and test as soon as I can, but please don't wait for me to merge if you feel good about it. Let's aim for Coursera integration in April.

**Comment by reginaastri** [comment]
Hi Nick, sounds like a plan. I merged the course already but adding corrections and improvements is no problem.

**Comment by ncarchedi** [comment]
@reginaastri Yes, I somehow didn't see the merge, but that's great. Will give you my thoughts/feedback as time permits. Thanks again!

**Comment by gauravksrivastava** [comment]
Using R Studio Version 0.98.1091 on OS X 10.10

Both install_from_swirl("Exploratory_Data_Analysis") and install_from_swirl("Exploratory Data Analysis") hang and the course does not get installed. 

And, efforts to install from github are not yielding (as illustrated below) too.

> install_course_github("reginaastri","Exploratory_Data_Analysis", multi=TRUE)
Error in unzip(path, list = TRUE) : 
  zip file '/Library/Frameworks/R.framework/Versions/3.1/Resources/library/swirl/Courses/temp.zip' cannot be opened
> install_course_github("reginaastri","Exploratory Data Analysis", multi=TRUE)
Error in unzip(path, list = TRUE) : 
  zip file '/Library/Frameworks/R.framework/Versions/3.1/Resources/library/swirl/Courses/temp.zip' cannot be opened

Any pointers?

**Comment by seankross** [comment]
@gauravksrivastava I've moved your issue to #140, let's continue this discussion there.
